tools/epub_reader.py

```diff
@@ -0,0 +1,169 @@
+                                    elif z.string:
+                                        temp = temp + stringify(z, 0)
+                                temp = temp + "\n"
+                            elif CurrSoup.find(class_="lines_float"):
+                                ChapCont = CurrSoup.find(class_="lines_float").contents
+
+                                temp = temp + "^"
+                                for z in ChapCont:
+                                    temp = temp + stringify(z, 0)
+                                temp = temp + "\n"
+                            elif CurrSoup.find(class_="event01"):
+                                ChapCont = CurrSoup.find(class_="event01").contents
+
+                                temp = temp + "<"
+                                for z in ChapCont:
+                                    temp = temp + stringify(z, 0)
+                                temp = temp + "\n"
+                            elif CurrSoup.find(class_="event1"):
+                                ChapCont = CurrSoup.find(class_="event1").contents
+
+                                for z in ChapCont:
+                                    temp = temp + stringify(z, 0)
+                                temp = temp + "\n"
+                            else:
+                                ChapCont = CurrSoup.contents[0].contents[0].contents
+
+                                for z in ChapCont:
+                                    temp = temp + stringify(z, 0)
+                                temp = temp + "\n"
+
+                        out.write(temp)
+            out.close()
+        i = i + 1
+
+
+
+def stringify(x, i):
+    curr = ""
+
+    if len(x) != 0:
+        for y in x:
+            curr = curr + str(y)
+
+    if x.string == x:
+        return str(x)
+    elif str(x) == '<span class="small">' + curr + '</span>':
+        temp = "__"
+
+        for y in x.contents:
+            temp = temp + stringify(y, i)
+        return temp + "__"
+    elif str(x) == '<em>' + curr + '</em>':
+        temp = "*"
+
+        for y in x.contents:
+            temp = temp + stringify(y, i)
+        return temp + "*"
+    elif str(x) == '<strong>' + curr + '</strong>':
+        temp = "**"
+
+        for y in x.contents:
+            temp = temp + stringify(y, i)
+        return temp + "**"
+    elif str(x) == '<sup class="frac">' + curr + '</sup>':
+        temp = ""
+
+        for y in x.contents:
+            temp = temp + stringify(y, i)
+        return "[^"+temp+"]"
+    elif str(x) == "<br/>":
+        if i == 0:
+            return "\n"
+        else:
+            j = 0
+            temp = ""
+
+            while j != i:
+                temp = temp + "#"
+                j += 1
+            return "\n" + temp
+    elif len(x.contents) != 1:
+        temp = ""
+
+        for y in x.contents:
+            temp = temp + stringify(y, i)
+        return temp
+    else:
+        return stringify(x.contents[0], i)
+
+
+_init_()
```

Comment:
```
if __name__ == "__main__":
    main()
```
This will only run when the script is called, not if it's imported by another script. You should probably use that here.